File: URI - Beecrowd/2455.py
p1, c1, p2, c2 = input().split()

esquerda = int(p1)*int(c1)
direita = int(p2)*int(c2)

if direita > esquerda:

  print(1)

elif esquerda > direita:

  print(-1)

elif esquerda == direita:

  print(0)

File: URI/2455.c
#include <stdio.h>

int main(void) {
  
  int p=0,c=0,p1=0,c1=0,esquerda=0,direita=0;

  scanf("%i %i %i %i", &p, &c, &p1, &c1);

  esquerda = p*c;
  direita = p1*c1;

  if(esquerda > direita){

    printf("-1\n");

  }

  else if(direita > esquerda){

    printf("1\n");
  }

  else if(direita == esquerda){

    printf("0\n");

  }

  return 0;
}